Sitenotice.php

```diff
@@ -27,20 +27,20 @@ function wfGlobalSiteNotice( &$siteNotice, $skin ) {
 
 		$siteNotice .= <<<EOF
 			<table style="width: 100%;">
-				<tbody><tr><td style="font-size: 120%; border-left: 4px solid #ff1e00; background-color: #ff5200cf; padding: 10px 15px; color: whitesmoke;">
+				<tbody><tr><td style="font-size: 120%; border-left: 4px solid #180F67; background-color: #F7F6FF; padding: 10px 15px;">
```

Comment:
This will revert it back to a colorless site notice though. The colors weren't changed because of Swift but because of my conversion with about 6 or 7 different users recieving feedback on it.
Which mostly agreed that the new colors were better to normalise it with the colorscheme we use throughout, like on the landing page.

I think for the change I also talked with you about it back the also?

Reply:
Didn't mean to "Request changes" here, this is just a comment we can decide on.

Reply:
I actually made this PR primarily because I have heard from users across the farm that the colors are a bit of an eyesore and tend to stick out. We could look into other colors but I chose the Meta pastel blue colors as blue is associated with information.

Reply:
I think we should just do the learn more button for now, as it is more important at the moment, and more time sensitive, but we can look into different colors later. If thats okay with you?

Reply:
That's fine